source/module_io/to_wannier90.cpp

```diff
@@ -1633,33 +1630,34 @@ std::complex<double> toWannier90::unkdotkb(const ModulePW::PW_Basis* rhopw,
 {
     // (1) set value
     std::complex<double> result(0.0, 0.0);
-    std::complex<double> *psir = new std::complex<double>[wfcpw->nmaxgr];
-    std::complex<double>* phase = new std::complex<double>[rhopw->nmaxgr];
+    std::complex<double>* psir = new std::complex<double>[wfcpw->nmaxgr];
+    std::complex<double>* phase = new std::complex<double>[wfcpw->nmaxgr];
+    ModuleBase::GlobalFunc::ZEROS(phase, wfcpw->nmaxgr);
```

Comment:
zeros psir too?

Reply:
phase is input
However, psir is output which is obtained from the function wfcpw->recip2real(&psi_pw(ik, iband_L, 0), psir, ik) and does not depend on any initial value.